components/automate-ui/src/app/modules/desktop/desktop-detail/desktop-detail.component.ts

```diff
@@ -53,6 +54,12 @@ export class DesktopDetailComponent implements OnInit {
     this.getCheckInHistory();
   }
 
+  ngOnDestroy() {
+    this.subscription.unsubscribe();
```

Comment:
This line is not needed; that's what the `isDestroyed.complete` handles automatically. 😁